lib/porky_lib/file_service.rb

```diff
@@ -91,6 +91,26 @@ def overwrite_file(file, file_key, bucket_name, key_id, options = {})
     tempfile.unlink
   end
 
+  def presigned_post_url(bucket_name, file_name, expires_in, ssekms_key_id, options = {})
+    obj = s3.bucket(bucket_name).object(file_name)
+
+    obj.presigned_url(:put,
```

Comment:
We may need to add more options here. I think the content type is not captured by s3, in my tests a text file was marked as binary/octet-stream